### DISCUSSION TITLE
Delete Collection?

### DISCUSSION THREAD

**Discussion by yaanahuu2** [arangodb/arangojs#747] [Ideas]
I can't find a way to drop/delete collections in the ArangoJS documentation.  Is there an implementation of 
DELETE /_api/collection/{collection-name} 

(https://www.arangodb.com/docs/stable/http/collection-creating.html#drops-a-collection)
in ArangoJS?  If not, is there a rationale for not having that capability?

Thanks,

Geoff

**Comment by ghost** [comment]
Does https://arangodb.github.io/arangojs/8.2.1/interfaces/collection.DocumentCollection.html#drop solve this problem?
